common/app/model/dotcomrendering/pageElements/CartoonExtraction.scala

```diff
@@ -0,0 +1,37 @@
+package model.dotcomrendering.pageElements
+
+import com.gu.contentapi.client.model.v1.{CartoonElementFields, CartoonImage}
+import model.ImageAsset
+
+object CartoonExtraction {
+  def extractCartoon(cartoonElementFields: CartoonElementFields): CartoonBlockElement = {
+    CartoonBlockElement(
+      variants = getCartoonVariants(cartoonElementFields),
+      role = Role(cartoonElementFields.role, Inline),
+      credit = cartoonElementFields.credit,
+      caption = cartoonElementFields.caption,
+      alt = cartoonElementFields.alt,
+      source = cartoonElementFields.source,
+      displayCredit = cartoonElementFields.displayCredit,
+    )
+  }
+
+  private def getCartoonVariants(cartoonData: CartoonElementFields): Option[List[DcrCartoonVariant]] = {
+    cartoonData.variants.map(maybeVariant =>
+      maybeVariant
+        .map(variant =>
```

Comment:
Minor thing, but the use of the word `maybeVariant` here perhaps suggests an `Option[Variant]` whereas it's actually a `List[Variant]`? Would this work as an alternative?

```suggestion
    cartoonData.variants.map(variants =>
      variants
        .map(variant =>
```